pkg/kapp/resources/identified_resources_pods.go

```diff
@@ -10,16 +10,17 @@ import (
 )
 
 func (r IdentifiedResources) PodResources(labelSelector labels.Selector) UniquePodWatcher {
-	return UniquePodWatcher{labelSelector, r.coreClient}
+	return UniquePodWatcher{labelSelector, r.resources.fallbackAllowedNamespaces, r.coreClient}
```

Comment:
instead of reaching into resources object, let's save off fallbackAllowedNamespaces onto IdentifiedResources directly and use it here: https://github.com/k14s/kapp/blob/0a2087f8225c9ea650c6244d71c3b5af20fbbfc2/pkg/kapp/resources/identified_resources.go#L21

Reply:
Thanks @cppforlife. Do you mean like this? Then fallbackAllowedNamespaces is duplicated with resources.fallbackAllowedNamespaces.

`return IdentifiedResources{coreClient, resourceTypes, resources, fallbackAllowedNamespaces, logger.NewPrefixed("IdentifiedResources")}`

Reply:
Or remove fallbackAllowedNamespaces from `resources` additionaly?